cime_config/acme/config_archive.xml

```diff
@@ -132,33 +134,33 @@
   <comp_archive_spec compname="dwav" compclass="wav">
     <rest_file_extension>\.r.*</rest_file_extension>
     <rest_history_varname>unset</rest_history_varname>
-    <rpointer_file>rpointer$NINST_STRING.wav</rpointer_file>
```

Comment:
It looks like these are changes that were not on the CESM side with this PR and were probably brought over when you copied the cesm file. Is this an error? Do we want these changes on ACME?